docs/geedocs/answer/4412441.html

```diff
@@ -186,6 +189,32 @@
 The new providers appear in the Provider Manager dialog in the order in which you added them:<br/>
 <br/>
 <img alt="Provider Manager" src="../art/fusion/tutorial/providerManager-full.png"/></li>
-<li>Click <strong>OK</strong> to close the Provider Manager, and go on to the <a href="../answer/4412443.html">next lesson</a>.</li>
-</ol></p><div class="footer"><p class="BackToTop"><a href="#top_of_file">Back to top</a> <hr /></p> <p class="copyright">&copy;2015 Google</p></div>
+
+<li>Click <strong>OK</strong> to close the Provider Manager.</li>
+</ol></p>
+
+<h2><a name="DefineResPath"></a>Define Resource Paths</h2>
+<p>The <strong>Asset Manager Resource Editor Default Source Paths</strong> screen allows you to specify default paths for Vector, Imagery, and Terrain resource types.</p>
+<h4>To define resource paths:</h4>
+<ol class="spaced-list">
+<li>Open the Asset Manager Resource Editor Default Source Paths by selecting <strong>Edit &gt; Preferences &gt; Asset Manager</strong>.<br/>
+<br/>
+<img alt="Asset Manager Config" src="../art/fusion/tutorial/assetManagerConfig.png"/></li>
+
+<li>Click the first folder icon on the right-hand side for Vector.<br/>
+<br/>
+<img alt="Folder Icon" src="../art/fusion/tutorial/iconProjBrowser.png"/><br/>
+<br/>
+The <strong>Select Folder</strong> dialog appears.<br/>
+<br/>
+<img alt="Select Folder Dialog" src="../art/fusion/tutorial/assetFolderSelection.png"/>
+</li>
+<li>
+Select <strong>Vector</strong>, then click <strong>OK</strong>.
```

Comment:
I think we should add a step before this step to navigate to /opt/google/share/tutorials/fusion in case the user doesn't start fusion in the tutorial directory.